pwndbg/prompt.py

```diff
@@ -5,17 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
+import subprocess
+
 import gdb
 
 import pwndbg.events
 import pwndbg.memoize
 import pwndbg.stdio
 
-msg = "Loaded %i commands.  Type pwndbg [filter] for a list." % len(pwndbg.commands._Command.commands)
-print(pwndbg.color.red(msg))
 
+def show_version():
+    try:
+        git_path = os.path.join(os.path.dirname(os.path.dirname(pwndbg.__file__)), '.git')
+        commit_id = subprocess.check_output(['git', '--git-dir', git_path, 'rev-parse', 'HEAD'])
+        commit_id = commit_id[:8].decode('utf-8')
```

Comment:
We should use `--short` instead of truncating to `8` characters, since 4 bytes of SHA is not guaranteed to be unique